src/submission/models.py

```diff
@@ -640,6 +643,17 @@ class Article(AbstractLastModifiedModel):
     language = models.CharField(max_length=200, blank=True, null=True, choices=LANGUAGE_CHOICES,
                                 help_text=_('The primary language of the article'))
     section = models.ForeignKey('Section', blank=True, null=True, on_delete=models.SET_NULL)
+    jats_article_type_override = DynamicChoiceField(max_length=255,
+                                                    dynamic_choices=get_jats_article_types(),
+                                                    choices=tuple(),
+                                                    blank=True, null=True,
+                                                    help_text="The type of article as per the JATS standard. The initial state of this field is set by the submission section's article type.",
```

Comment:
This help text needs adjusting now, the initial state is no longer set by the article type.

"An override of the article type when encoding into JATS. If the article has a section, it will use it as its article type"

Reply:
Done